kyuubi-server/src/test/scala/org/apache/kyuubi/credentials/HiveDelegationTokenProviderSuite.scala

```diff
@@ -93,7 +93,8 @@ class HiveDelegationTokenProviderSuite extends KerberizedTestHelper {
     FileUtils.deleteDirectory(hadoopConfDir)
   }
 
-  test("obtain hive delegation token") {
+  // Ignore the test because LocalMetaServer can not work with Thrift 0.16.0.
+  ignore("obtain hive delegation token") {
```

Comment:
cc @zhouyifan279 
We can not start `HiveMetastore` using thrift 0.16.0, because the Interface method signature changed, the class is not compatible.

Reply:
I will create a PR later to fix it.

Reply:
> Shall we govern these ignored tests with flaky tests together? cc @yanghua

I suggest we split them into two categories. The ignored tests may have other reasons not only flaky tests?